packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -386,6 +386,25 @@ backbone = keras_nlp.models.Backbone.from_preset("hf://${model.id}")
 `,
 ];
 
+export const keras_hub = (model: ModelData): string[] => [
+	`# Available backend options are: "jax", "torch", "tensorflow".
+import os
+os.environ["KERAS_BACKEND"] = "jax"
+
+import keras_hub
+
+# Load a task-specific model (*replace CausalLM with your task*)
+model = keras_hub.models.CausalLM.from_preset("hf://${model.id}", dtype="bloaf16)
+
+# Possible tasks are CausalLM, TextToImage, ImageClassifier, ...
+# full list here: https://keras.io/api/keras_hub/models/#api-documentation
+
+# Model components can also be laoded separately:
+tokenizer = keras_hub.models.Tokenizer.from_preset("hf://${model.id}")
+backbone = keras_hub.models.Backbone.from_preset("hf://${model.id}")
```

Comment:
maybe not worth adding, maybe just the main arch/task is better/simpler

Reply:
sure, that works too.